Jenkinsfile

```diff
@@ -13,14 +13,15 @@ pipeline {
           branch 'PR-*'
         }
         environment {
-          PREVIEW_VERSION = "0.0.0-SNAPSHOT-$BRANCH_NAME-$BUILD_NUMBER"
+          PREVIEW_VERSION = "7.1.0-$BRANCH_NAME-$BUILD_NUMBER"
           PREVIEW_NAMESPACE = "$APP_NAME-$BRANCH_NAME".toLowerCase()
           HELM_RELEASE = "$PREVIEW_NAMESPACE".toLowerCase()
         }
         steps {
           container('maven') {
             sh "mvn versions:set -DnewVersion=$PREVIEW_VERSION"
             sh "mvn install -DskipITs"
+            sh "mvn deploy -DskipTests -DskipITs"
```

Comment:
Is this change and the previous one suppose to be merged or it's just temporary?

Reply:
Fixed in 9e13aa5029b819a4cf241315888770760f02e5b4